mobile/src/reducers/user/user-preferences.model.ts

```diff
@@ -0,0 +1,40 @@
+import { dataMember, required } from 'santee-dcts';
+
+export class UserPreferences {
+
+    @dataMember()
+    @required()
+    public emailNotifications: boolean;
```

Comment:
Are you sure you want to use boolean here? And if yes, could you please consider using Optional<boolean>? Cause using 'false' as initial value looks like a quite bad idea.

Reply:
For me it's better to make UserPreferences nullable in default UserInfoState. Don't you think so?

Reply:
68ee907a2dc9b8b30b1a5b4afc4821ada7e05cd9

Reply:
Let be Optional.